mmseg/datasets/custom.py

```diff
@@ -99,6 +99,9 @@ def __init__(self,
         self.label_map = None
         self.CLASSES, self.PALETTE = self.get_classes_and_palette(
             classes, palette)
+        if test_mode:
+            assert self.CLASSES is not None, \
+                '`cls.CLASSES` or `classes` should be specified when testing'
```

Comment:
It seems that this modify leads to failed github CI (checked).
Could you please add some unittests and fix the failed unitsests ?

Reply:
https://github.com/open-mmlab/mmsegmentation/pull/833/checks?check_run_id=3449373682

Reply:
https://github.com/open-mmlab/mmsegmentation/pull/833/checks?check_run_id=3449373726

Reply:
OK, I will find time to fix the issue above 😂